crates/extricrate/Cargo.toml

```diff
@@ -11,6 +11,9 @@ license.workspace = true
 repository.workspace = true
 
 [dependencies]
+quote = "1.0.40"
+syn = { version = "2.0.101", features = ["extra-traits", "full", "parsing", "printing", "visit"] }
```

Comment:
Doesn't full imply the rest?

Reply:
I removed some of the extra features, but for example `full` doesn't imply `visit` apparently so I've left that one